.env

```diff
@@ -97,7 +97,9 @@ DEVTOOLSET_VERSION=
 # actions cache up to date for the macOS wheels:
 #   https://github.com/ursacomputing/crossbow/blob/master/.github/workflows/cache_vcpkg.yml
 # vcpkg minimum version "09adfdc8cdad76345b7cc7f3305899e1cbd66297" due to CVE-2022-3786
-VCPKG="2871ddd918cecb9cb642bcb9c56897f397283192"
+# VCPKG="a7b6122f6b6504d16d96117336a0562693579933"  # 2023.02.24 release version
```

Comment:
Could you remove this?
This is needless, right?

Reply:
I have removed it